lib/routes.jsx

```diff
@@ -115,6 +115,8 @@ var routeElements = Object.keys(pages).map(function(path) {
   return <Route path={path} component={pages[path]} key={path}/>;
 });
 
+var makerPartyLocales = ['de', 'fr', 'es', 'pl', 'it', 'nl', 'cs', 'bg'];
```

Comment:
Based on:

Tier 1: English, French, German, Spanish, Polish

Tier 2: Italian, Dutch, Czech, Bulgarian, 

from https://github.com/MozillaFoundation/Advocacy/issues/151

it looks right to me.5 primary (including English)

and 4 secondary